client/components/Home/index.tsx

```diff
@@ -1,32 +1,85 @@
-import React, { ReactElement } from "react";
-import Typography from "@material-ui/core/Typography";
-
+/* eslint-disable prettier/prettier */
+import React, { ReactElement, useState } from "react";
+import { Link } from "react-router-dom";
+import Axios from "axios";
 import { useHomeStyles } from "./styles";
 
-import Feed from "./Feed";
-import RecentVideos from "./RecentVideos";
-import Film from "./FeaturedMovie";
-
 const Home = (): ReactElement => {
   const classes = useHomeStyles({});
+  const [item, setItem] = useState(null);
+  const [refresh, setRefresh] = useState(true);
+  const [itemReady, setItemReady] = useState(false);
+
+  const getRecommendation = (): void => {
+    const list: Array<object> = [];
+    for (let index = 0; index < 4; index++) {
+      Axios(`https://tv-v2.api-fetch.website/movies/1`).then((res) => {
+        const id = Math.round(Math.random() * 50);
+        console.log(id);
+        list.push({
+          title: res.data[id].title,
+          img: res.data[id].images.poster,
+          imdbId: res.data[id].imdb_id,
+          genres: res.data[id].genres,
+          src: `http://localhost:8080/movie/${res.data[id].imdbId}`
```

Comment:
```suggestion
          src: `http://localhost:8080/movie/${res.data[id].imdb_id}`
```
Shouldn't it be like this instead ? Yours results in `src: "http://localhost:8080/movie/undefined"`